internal/service/ec2/vpc_test.go

```diff
@@ -593,6 +594,61 @@ func TestAccVPC_assignGeneratedIPv6CIDRBlock(t *testing.T) {
 	})
 }
 
+func TestAccVPC_assignGeneratedIPv6CIDRBlockWithBorder(t *testing.T) {
+	var vpc ec2.Vpc
+	resourceName := "aws_vpc.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.PreCheck(t) },
+		ErrorCheck:   acctest.ErrorCheck(t, ec2.EndpointsID),
+		Providers:    acctest.Providers,
+		CheckDestroy: testAccCheckVpcDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVpcConfigAssignGeneratedIpv6CidrBlockWithBorder(true),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					acctest.CheckVPCExists(resourceName, &vpc),
+					testAccCheckVpcCidr(&vpc, "10.1.0.0/16"),
+					resource.TestCheckResourceAttr(resourceName, "assign_generated_ipv6_cidr_block", "true"),
+					resource.TestCheckResourceAttr(resourceName, "cidr_block", "10.1.0.0/16"),
+					resource.TestCheckResourceAttr(resourceName, "ipv6_cidr_block_network_border_group", "us-west-2-lax-1"),
```

Comment:
can you make ` "us-west-2-lax-1"` a variable in this function and pass it to both your Config function (hcl) and this test check

the syntax to have multiple interpolated values is like:
```
  tags = {
    %[1]q = %[2]q
  }
  ```

if youre trying to reuse the same function for enabled / disabled, terraform parameters can accept `null` to omit

Reply:
Done